src/NuGet.Services.BasicSearch/Web.config

```diff
@@ -51,7 +51,10 @@
     <validation validateIntegratedModeConfiguration="false"/>
     <modules>
       <remove name="WebDAVModule"/>
+      <remove name="TelemetryCorrelationHttpModule"/>
+      <add name="TelemetryCorrelationHttpModule" type="Microsoft.AspNet.TelemetryCorrelation.TelemetryCorrelationHttpModule, Microsoft.AspNet.TelemetryCorrelation" preCondition="integratedMode,managedHandler"/>
       <remove name="ApplicationInsightsWebTracking"/>
+      <add name="ApplicationInsightsWebTracking" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" preCondition="managedHandler"/>
```

Comment:
Not sure whether this was on purpose or not, but old search didn't have AI Web Tracking enabled?

Reply:
Does web tracking mean the JavaScript side?

Reply:
The Application Insights settings are read by the `ApplicationInsightsHttpModule`, which wasn't configured in this `web.config` file, which leads me to think the `ApplicationInsights.config` file was completely ignored.